regtest.py

```diff
@@ -546,7 +546,10 @@ def test_suite(argv):
             else:
                 comp_string, rc = suite.build_c(test=test, outfile=coutfile)
 
-            executable = test_util.get_recent_filename(bdir, "", ".ex")
+            if test.run_as_script:
+                executable = None
+            else:
+                executable = test_util.get_recent_filename(bdir, "", ".ex")
```

Comment:
Should also be `None` for `test.customRunCmd`.

I can check again, but I think this returns `None` anyway if not found. So let's revert this change:
```suggestion
            executable = test_util.get_recent_filename(bdir, "", ".ex")
```